outputs.tf

```diff
@@ -57,6 +62,16 @@ output "cos_instance_guid" {
   value       = local.cos_instance_guid
 }
 
+output "cos_instance_name" {
+  description = "The name of the Cloud Object Storage Instance where buckets are created"
```

Comment:
remove "where buckets are created" - this module supports creating only the instance

`Instance` should be lowercase `instance`